### PR TITLE
Greatly increase the amount of logging information.

### PR DESCRIPTION
**Pull request by oddstr13**
The main change so far is to include file paths and line numbers on log output.

Log all calls to `__getitem__` and `__shortcuts__` (`DEBUG`, noisy)

Some flake8 errors corrected.

Currently prints some stacktraces that probably shouldn't be printed, but my opinion on that is to keep it like that until exception handling can be improved (and made much more specific) in those areas.
```log
00:16:39.614 T:140707650131712  NOTICE: JELLYFIN.database.jellyfin_db -> ERROR::resources/lib/database/jellyfin_db.py:161 'NoneType' object has no attribute '__getitem__'
                                            Traceback (most recent call last):
                                              File "resources/lib/database/jellyfin_db.py", line 159, in get_version
                                                version = self.cursor.fetchone()[0]
                                            TypeError: 'NoneType' object has no attribute '__getitem__'
```

Also added editorconfig.

### CODE REVIEW

File: resources/lib/player.py [hunk 1]
```diff
@@ -1,447 +1,447 @@
-# -*- coding: utf-8 -*-
```
Comment (oddstr13) [inline resources/lib/player.py]: It seems this file had Windows file endings. Long live editorconfig?

File: resources/lib/jellyfin/core/api.py [hunk 2]
```diff
@@ -334,6 +334,3 @@ def close_transcode(self, device_id):
         return self._delete("Videos/ActiveEncodings", params={
             'DeviceId': device_id
         })
-
-    def delete_item(self, item_id):
-        return self.items("/%s" % item_id, "DELETE")
```
Comment (oddstr13) [inline resources/lib/jellyfin/core/api.py]: flake8 was kind enough to informing me that this was a duplicate definition.
You should be able to find the exact same code further up.

File: resources/lib/jellyfin/core/configuration.py [hunk 3]
```diff
@@ -38,14 +40,16 @@ def __shortcuts__(self, key):
...
         return self.data.get(key, self.__shortcuts__(key))
 
     def __setitem__(self, key, value):
         self.data[key] = value
 
     def app(self, name, version, device_name, device_id, capabilities=None, device_pixel_ratio=None):
-        
         LOG.info("Begin app constructor.")
+        # import traceback
+        # LOG.debug(''.join(['\n'] + traceback.format_stack()))
```
Comment (cvium) [inline resources/lib/jellyfin/core/configuration.py]: Remove or uncomment

Comment (oddstr13) [reply resources/lib/jellyfin/core/configuration.py]: Right, that's from attempting to trace down what's happening with the `config['app.name']` being `None` exception.
I didn't want to lose it, so I commented it out in stead of removing. Fixed now.

**Pull request merged**